Formula/git-filter-repo.rb

```diff
@@ -15,7 +15,7 @@ class GitFilterRepo < Formula
   #  * Don't use git as a dependency (it's always available)
   # But we require Git 2.22.0+
   # https://github.com/Homebrew/homebrew-core/pull/46550#issuecomment-563229479
-  depends_on "git"
+  depends_on "git" if MacOS.version <= :mojave
```

Comment:
I think this can be
```suggestion
  uses_from_macos "git", since: :catalina
```

Reply:
Hmm, I thought `uses_from_macos` is noop on macOS

Reply:
We can also remove the comment above, since the audit only fails for new formulae.

Reply:
Note that both versions (before and after my suggestion) will require `git` on Linux. That's probably what we want, but just putting it out there to be sure.

Reply:
Yep, thanks, this should work.

Reply:
Nope, I lied
```
$ brew audit --strict --fix git-filter-repo
git-filter-repo:
  * 15: col 3: `uses_from_macos` should only be used for macOS dependencies, not git.
Error: 1 problem in 1 formula detected
```

Reply:
oops

Reply:
Looks like I misled you with my review. You can blame @SMillerDev. I wouldn't be here if I wasn't told to do some 😄

Reply:
It looks like `git` should be added to `ALLOWED_USES_FROM_MACOS_DEPS` list https://github.com/Homebrew/brew/blob/9657303ed2c484a4bd8f7527706940f9e4d43354/Library/Homebrew/rubocops/uses_from_macos.rb#L73-L92

Reply:
> It looks like `git` should be added to `ALLOWED_USES_FROM_MACOS_DEPS` list https://github.com/Homebrew/brew/blob/9657303ed2c484a4bd8f7527706940f9e4d43354/Library/Homebrew/rubocops/uses_from_macos.rb#L73-L92

https://github.com/Homebrew/brew/pull/11506